demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -817,13 +819,14 @@ export class DemoMeetingApp
       this.log('Video input quality is changed');
       switch (videoInputQuality.value) {
         case '360p':
-          this.audioVideo.chooseVideoInputQuality(640, 360, 15, 600);
+          this.audioVideo.chooseVideoInputQuality(640, 360, 15);
+          this.audioVideo.setVideoMaxBandwidthKbps(600);
           break;
         case '540p':
-          this.audioVideo.chooseVideoInputQuality(960, 540, 15, 1400);
+          this.audioVideo.chooseVideoInputQuality(960, 540, 15);
```

Comment:
Do we need to `this.audioVideo.setVideoMaxBandwidthKbps()` for 540p and 720p as well?